kotlin/Dynamic Programming/LongestPalindromInAStringExpanding.kt

```diff
@@ -0,0 +1,35 @@
+fun longestPalindromeInAStringExpanding(s: String): String {
```

Comment:
Please fix typo in file name

Reply:
Updated by this commit - https://github.com/ByteByteGoHq/coding-interview-patterns/pull/43/commits/b306f226e0d2113c25be443239d0976612cd39e1